Documentation/data-collection.md

```diff
@@ -236,6 +236,10 @@ data:
     - '{__name__="che_workspace_start_time_seconds_sum"}'
     # (codeready workspaces, @ibuziuk) The overall number of attempts for starting all the workspaces.
     - '{__name__="che_workspace_start_time_seconds_count"}'
+    # (image-registry-team) apiserver_v1_image_imports_total counts the number of image stream
+    # imports made using registry protocol v1, we are moving forward with deprecating this version
+    # hence we need to know how often it is used.
+    - '{__name__="apiserver_v1_image_imports_total"}'
```

Comment:
Is this by default in every openshift, as I did not see it on my cluster?

Reply:
Yes it is by default in all clusters. It will show up as soon as you import an image from a image registry using protocol v1. This protocol version has already been deprecated by docker and we don't expect this metric to show up.

Reply:
@lilic will it expose the metric labels? We need only sum(apiserver_v1_image_imports_total), labels may have sensitive data (names of private registries and repositories).

Reply:
You have to create a recording rule doing the sum() then.

Reply:
@simonpasquier I have sent over a new commit that adds `:sum` at the end of the metric we are querying. Could you please confirm this is the right way of achieving the desired outcome?

Reply:
Have you defined the recording rule that generates the `apiserver_v1_image_imports:sum` somewhere (typically in the operator managing the component emitting the `apiserver_v1_image_imports_total` metric)?

Reply:
@simonpasquier no, could you please point me to a sample where this is done?

Reply:
@simonpasquier I have created the rule in here, could you take a look? https://github.com/openshift/cluster-image-registry-operator/pull/626